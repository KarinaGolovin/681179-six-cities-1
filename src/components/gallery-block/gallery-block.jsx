import React from 'react';
import PropTypes from 'prop-types';

export const GalleryBlock = ({images}) => {
  return (
    <div className="property__gallery-container container">
      <div className="property__gallery">
        {images.map((image) => {
          return (
            <div className="property__image-wrapper" key={image}>
              <img className="property__image" src={image} alt="Photo studio"/>
            </div>
          );
        })}
      </div>
    </div>
  );
};

GalleryBlock.propTypes = {
  images: PropTypes.arrayOf(PropTypes.string)
}
