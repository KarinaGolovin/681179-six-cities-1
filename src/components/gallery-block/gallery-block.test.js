import React from 'react';
import renderer from 'react-test-renderer';
import {GalleryBlock} from './gallery-block.jsx';

const mock = [];

it(`GalleryBlock correctly renders`, () => {
  const component = renderer.create(
      <GalleryBlock
        images={mock}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});
