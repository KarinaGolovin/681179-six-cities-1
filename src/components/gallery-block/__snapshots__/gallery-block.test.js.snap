// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryBlock correctly renders 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  />
</div>
`;
